lib/api_callers/json_request.rb

```diff
@@ -1,15 +1,10 @@
 require 'api_callers/http_request'
+require 'json'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.